extensions/tmtv/src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { id } from './id';
 import getHangingProtocolModule from './getHangingProtocolModule';
+import getPanelModule from './getPanelModule';
```

Comment:
I wonder if this should just be a generic pet-suv module rather than named the `tmtv` extension? The hanging protocol and side panel could be re-used in other modes, no?

Reply:
Yes I agree, pet-suv will be more understood and get a better audiance than tmtv.